### PR TITLE
Todolist app

### PR DESCRIPTION
**Pull request by aberllin**
Created Todo app and add styles 
Netlify - https://deploy-preview-4--optimistic-fermat-7ff6ec.netlify.app/

### CODE REVIEW

**Comment by aberllin** [comment]
https://deploy-preview-4--optimistic-fermat-7ff6ec.netlify.app/

File: public/index.html [hunk 1]
```diff
@@ -15,6 +15,7 @@
       user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
     -->
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+    <link rel="stylesheet" href='../src/index.css'>
```
Comment (ruslanakholod) [inline public/index.html]: why do we need it?

File: src/components/apps/ToDoApp/Form.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+export const Form = ({ inputText, setInputText, todos, setTodos }) => {
+  const inputTextHandler = (e) => {
+    setInputText(e.target.value)
+  }
+
+  const submitTodoHandler = (e) => {
+    e.preventDefault()
+    setTodos([...todos, { text: inputText, id: Date.now(), complete: false }])
+    setInputText('')
+    console.log(e)
```
Comment (ruslanakholod) [inline src/components/apps/ToDoApp/Form.js]: delete it

File: src/components/apps/ToDoApp/Form.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+  }
+
+  return (
+    <FormWrapper>
+      <StyledInput
+        value={inputText}
+        type="text "
```
Comment (ruslanakholod) [inline src/components/apps/ToDoApp/Form.js]: type="text"

**Comment by ruslanakholod** [comment]
image

If the user doesn't enter anything in the input, disable button

**Comment by ruslanakholod** [comment]
image
disable to enter spaces if there is no text before

**Comment by ruslanakholod** [comment]
image
fix styles

**Comment by ruslanakholod** [comment]
image
fix this please

File: public/index.html (hunk 1 above)
Comment (aberllin) [reply public/index.html]: I fixed it, there is no reason to use, I use Global Styles from styled-components

File: src/components/apps/ToDoApp/Form.js (hunk 2 above)
Comment (aberllin) [reply src/components/apps/ToDoApp/Form.js]: fixed

File: src/components/apps/ToDoApp/Form.js (hunk 3 above)
Comment (aberllin) [reply src/components/apps/ToDoApp/Form.js]: fixed

**Comment by aberllin** [comment]
> image
> 
> If the user doesn't enter anything in the input, disable button

fixed

**Comment by aberllin** [comment]
> image
> fix styles

fixed

**Comment by aberllin** [comment]
> image
> disable to enter spaces if there is no text before

fixed

**Comment by arevej** [comment]
Once you fix any of the comments, just react to the comment with 👍

File: src/components/apps/ToDoApp/Form.js [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+      setInputText(e.target.value)
+    }
+  }
+
+  const submitTodoHandler = (e) => {
+    e.preventDefault()
+    if (inputText === '') {
+      return ''
+    } else {
+      setTodos([...todos, { text: inputText, id: Date.now(), complete: false }])
```
Comment (arevej) [inline src/components/apps/ToDoApp/Form.js:19]: Nice that you decided to have ids and found a way to make them unique!

File: src/components/apps/ToDoApp/Todo.js [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+    )
+  }
+
+  const deleteHandler = () => {
+    setTodos(todos.filter((el) => el.id !== todo.id))
+  }
+
+  return (
+    <TodoWrapper>
+      <CompleteButton todo={todo} onClick={completeHandler}>
```
Comment (arevej) [inline src/components/apps/ToDoApp/Todo.js:24]: Would be nice to have a different icon next to the completed tasks e.g. filled

Comment (arevej) [inline src/components/apps/ToDoApp/Todo.js:25]: Would be nice to have a different icon next to the completed tasks e.g. filled

**Review by arevej — APPROVED** [review]

**Pull request merged**